components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -0,0 +1,331 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCActionSheetController.h"
+#import "MDCActionSheetItemView.h"
+#import "MDCActionSheetListViewController.h"
+#import "MaterialBottomSheet.h"
+#import "MaterialApplication.h"
+#import "MaterialTypography.h"
+
+//NSString *const kTableReuseIdentifier = @"BaseCell";
+
+@interface MDCActionSheetAction ()
+
+@property(nonatomic, nullable, copy) MDCActionSheetHandler completionHandler;
+
+@end
+
+@implementation MDCActionSheetAction
+
++ (instancetype)actionWithTitle:(nonnull NSString *)title
+                          image:(nullable UIImage *)image
+                          handler:(void (^__nullable)(MDCActionSheetAction *action))handler {
+    return [[MDCActionSheetAction alloc] initWithTitle:title
+                                                 image:image
+                                               handler:handler];
+}
+
+- (instancetype)initWithTitle:(nonnull NSString *)title
+                        image:(nullable UIImage *)image
```

Comment:
I don't believe we need to re-signify the nullability annotations in the Objective-C .m files.

Reply:
Done